artifacts/fedora/fedora.go

```diff
@@ -68,18 +69,21 @@ func (f *fedora) Inspect() (*api.ArtifactDetails, error) {
 	}
 
 	for i, release := range releases {
-		if f.releaseMatches(&releases[i]) {
-			components := strings.Split(release.Link, "/")
-			fileName := components[len(components)-1]
-			additionalTag := strings.TrimSuffix(strings.TrimPrefix(fileName, "Fedora-Cloud-Base-"), ".x86_64.qcow2")
-
-			return &api.ArtifactDetails{
-				SHA256Sum:            release.Sha256,
-				DownloadURL:          release.Link,
-				AdditionalUniqueTags: []string{additionalTag},
-				ImageArchitecture:    "amd64",
-			}, nil
+		if !f.releaseMatches(&releases[i]) {
```

Comment:
nit - unrelated optimisation

Reply:
Sorry, why do you mean? I added `continue` because I think `make lint` complained about it.

Reply:
Odd that lint suddently triggered on this?

My point was that it isn't a required in the wider scope of your change. While useful an optimisation like this can be done in a follow up.